### PR TITLE
Fix PG corrupt alerting tests

### PR DESCRIPTION
**Pull request by fbalak**

### CODE REVIEW

File: tests/manage/monitoring/conftest.py [hunk 1]
```diff
@@ -341,10 +341,12 @@ def teardown():
     logger.info(f"Put object into {pool_name}")
     pool_object = "test_object"
     ct_pod.exec_ceph_cmd(f"rados -p {pool_name} put {pool_object} /etc/passwd")
+    logger.info(f"Corrupting pool {pool_name} on {osd_deployment.name}")
+    rados_utils.corrupt_pg(osd_deployment, pool_name, pool_object)
 
     def corrupt_pg():
```
Comment (DanielOsypenko) [inline tests/manage/monitoring/conftest.py:347]: to leave the internal function corrupt_pg() which will now sleep and return osd_deployment.name will be misleading as we use it as an 'opetation' for measure_operation call. May be rename it? or simplify and remove it? WDYT?

Comment (fbalak) [reply tests/manage/monitoring/conftest.py]: Good point, name of the function and docstring were changed.

File: ocs_ci/ocs/rados_utils.py [hunk 2]
```diff
@@ -285,46 +285,43 @@ def corrupt_pg(osd_deployment, pool_name, pool_object):
...
-    osd_containers = osd_data["spec"]["containers"]
-    original_osd_cmd = " ".join(osd_containers[0].get("command"))
-    original_osd_args = osd_containers[0].get("args")
-    original_osd_args = [",".join(arg.split()) for arg in original_osd_args]
-    original_osd_args.remove("--foreground")
-    original_osd_args = " ".join(original_osd_args)
-    logger.info(f"Original args for osd deployment: {original_osd_args}")
     osd_id = osd_data["metadata"]["labels"]["ceph-osd-id"]
 
+    for i_container in osd_data["spec"]["initContainers"]:
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/rados_utils.py:290]: please take it just as a nice syntax feature, but not necessary thing.
This block might look also like this:
"""
bluefs_container = None
for i_container in osd_data["spec"]["initContainers"]:
    if i_container["name"] == "expand-bluefs":
        bluefs_container = i_container
        break
else:
    raise ValueError("expand-bluefs container is missing")
"""

**Review by DanielOsypenko — APPROVED** [review]

File: ocs_ci/ocs/rados_utils.py (hunk 2 above)
Comment (fbalak) [reply ocs_ci/ocs/rados_utils.py:291]: Looks good

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/25829/

**Review by petr-balogh — APPROVED** [review]

**Review by DanielOsypenko — APPROVED** [review]

**Pull request merged**